scRNA-seq-advanced/01-read_filter_normalize_scRNA-live.Rmd

```diff
@@ -113,6 +111,8 @@ Some of these functions for other common data formats are discussed in [Chapter
 ```{r read SCE, live=TRUE}
 # read SCE from matrix directory
 
+  # ensure barcodes are set as column names in the SCE object
+
```

Comment:
@jashapiro would you prefer this comment not be there (and I move it to the end of the line so it goes away) for when you teach this? I'm going to fix those spots I introduced in the integration notebook; let me know if you want me to fix this one too.

Reply:
yeah, probably better to have it at the end of the line